### ISSUE TITLE
环境问题（是否要求linux系统以及python版本）

### ISSUE THREAD

**Issue by stuwrc** [wenyyu/Image-Adaptive-YOLO#62]
作者您好，请问作者这个工程是否一定要求Linux系统，我用Windows10系统按照requirments要求安装了环境，但是运行时报错显示numpy版本过低（目前是numpy==1.15.1,python3.7),感谢您的帮助。
ModuleNotFoundError: No module named 'numpy.core._multiarray_umath'
ImportError: numpy.core.multiarray failed to import
ImportError: numpy.core._multiarray_umath failed to import
ImportError: numpy.core.umath failed to import

**Comment by Mr-Hu123** [comment]
win10可以跑的，我就是用的win10，自测numpy==1.16.1是可以运行的，你可以把numpy升到这个版本

**Comment by stuwrc** [comment]
感谢您的帮助

**Comment by Kiumb1223** [comment]
My compile configuration is that python == 3.7 && numpy == 1.16.1 && protobuf == 3.20.0

And there is [a discussion in stackoverflow](https://stackoverflow.com/questions/54665842/when-importing-tensorflow-i-get-the-following-error-no-module-named-numpy-cor) about the problem you have proposed.

BTW,it is important to know that the path separator returned by `os.path.join` depends on the OS you\`re using. Cuz  on UNIX and Linux ,`os.path.join` will use `/` as the separator , and on windows .`os.path.join` will use `\` as the separator . So when running `evaluate.py` on win10 , there are likelihood that the project cann`t output the result image.

So how to run `evaluate.py` correctly? My solution is to  modify the code on line 199 like this:
```python
if os.name == 'posix':
  # Linux/Unix分割符 是 /
  image_name = image_path.split('/')[-1]
elif os.name == 'nt':
  # Windows   分割符 是 \
  image_name = image_path.split('\\')[-1]
```

**Comment by 7W7W7W** [comment]
@Kiumb1223 yes,that is very importent on windows! i lost lots of time until i see the issue.thanks!
